molecule/default/prepare.yml

```diff
@@ -1,18 +1,23 @@
 ---
 - name: Prepare
   hosts: all
+  become: true
   tasks:
     - name: Install required packages
       yum:
         use_backend: dnf
```

Comment:
I'm curious.. Why using "backend dnf" instrad of directly ansible.builtin.dnf ? And btw, the ansible.builtin is missing here :)